src/components/sections/campaign/AllActionsPane.jsx

```diff
@@ -82,8 +83,10 @@ export default class AllActionsPane extends CampaignSectionPaneBase {
         else if (actionList.items.length == 0) {
             endDate = startDate.clone().addDays(8 * 7);
         }
-
-        if (this.state.viewMode == 'cal') {
+        if (actionList.isPending) {
```

Comment:
This will never happen, because on line 66, an `actionList` without `items` will prevent rendering altogether.